content/blog/2019/08/2019-08-22-devops-world.adoc

```diff
@@ -0,0 +1,106 @@
+---
+layout: post
+title: "DevOps World - Jenkins World 2019 Experience"
```

Comment:
Maybe makes sense to add something to the title to make it a bit more personalized.

```suggestion
title: "My DevOps World - Jenkins World 2019 Experience"
```

As a quick patch, but maybe something more

Reply:
I love how it emphasizes that it's a personnal experience as a community member